.gitlab-ci.yml

```diff
@@ -33,7 +33,7 @@ variables:
 
 .docker-cache-status:              &docker-cache-status
   variables:
-    CARGO_HOME:                    "/cargo/${CI_JOB_NAME}"
+    CARGO_HOME:                    "/ci-cache/parity-ethereum/cargo/${CI_JOB_NAME}"
```

Comment:
This will be needed to use shared runners, 'substrate' will use `"/ci-cache/substrate/cargo/${CI_JOB_NAME}"` dir.
Should make changes in `volumes` runners configs.

Reply:
perfect